net/net.sh

```diff
@@ -61,7 +61,7 @@ Operate a configured testnet
                                       - Override the default --hashes-per-tick for the cluster
    --no-airdrop
                                       - If set, disables airdrops.  Nodes must be funded in genesis config when airdrops are disabled.
-   --lamports NUM_LAMPORTS_TO_MINT
+   --mint-lamports NUM_LAMPORTS_TO_MINT
```

Comment:
What do you think about using this opportunity to rename this from `mint` to `faucet`?  It doesn't mint tokens. It's just used to airdrop existing tokens.

Reply:
I don't feel strong either way.   "mint" now is just an extra account with some tokens.

Reply:
I'm renaming to faucet, it is a little more clear.

Reply:
Alright, fine.  Originally, mint meant "the first owner of all the tokens". Now it means "owner of the faucet tokens, which are delivered by a thing called a drone, that's only called a drone and not a faucet because the drone code was never generalized to the thing the book calls a drone. 
 "https://docs.solana.com/book/v/master/programs/drones